examples/rl_order_execution/README.md

```diff
@@ -16,7 +16,6 @@ To run codes in this example, we need data in pickle format. To achieve this, ru
 
 ```
```

Comment:
Please leave TODO for the guidance of future design.
```
Instead of dumping dataframe with different format(like `_gen_dataset` and `_gen_day_dataset` in   `qlib/contrib/data/highfreq_provider.py`),  we encournage to implement different subclass of `Dataset` and `DataHandler`.    This will keep the workflow cleaner and interfaces more consistent. and move all the compexity to the subclass.
```